mods/persistence/modules/clothing/_clothing.dm

```diff
@@ -0,0 +1,19 @@
+/obj/item/clothing/Initialize()
+	// Call on_attached on each accessory to allow for the addition of verbs etc. 
+	if(persistent_id)
+		if(islist(starting_accessories))
+			starting_accessories.Cut()
+		if(length(accessories))
+			for(var/obj/item/clothing/accessory/A in accessories)
+				A.on_attached(src, null)
+				if(A.removable)
+					src.verbs |= /obj/item/clothing/proc/removetie_verb
```

Comment:
That's oddly specific to remove the removetie_verb in a generic parent class?
Maybe this could be moved to the accessory's implementation?

Reply:
Despite the name, the removetie verb is actually generic for removing *all* accessories. For reference most of this behavior is taken from the ``add_accessory`` proc on ``/obj/item/clothing``

Reply:
Oh wow.. Alright then.